src/snowflake/snowpark/modin/plugin/docstrings/base.py

```diff
@@ -2048,6 +2048,90 @@ def reorder_levels():
     def resample():
         """
         Resample time-series data.
+
+        Convenience method for frequency conversion and resampling of time series.
+        The object must have a datetime-like index (DatetimeIndex, PeriodIndex, or TimedeltaIndex),
+        or the caller must pass the label of a datetime-like series/index to the ``on``/``level`` keyword parameter.
+
+        Parameters
+        ----------
+        rule : DateOffset, Timedelta or str
+            The offset string or object representing target conversion.
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+            Which axis to use for up- or down-sampling. For Series this parameter is unused and defaults to 0.
+            Must be DatetimeIndex, TimedeltaIndex or PeriodIndex.
```

Comment:
Do we natively support DatetimeIndex and the other specialized index classes?

Reply:
This is straight from pandas docs, put what we support in the Notes section. Is that not how it should be done?

Reply:
We're not super consistent about where we put divergent/unimplemented behavior, some methods like [GroupBy.transform](https://docs.snowflake.com/en/LIMITEDACCESS/snowpark-pandas-api/reference/modin/pandas_api/snowflake.snowpark.modin.pandas.groupby.DataFrameGroupBy.transform.html) keep it inline with the parameter, while others like [DataFrame.join](https://docs.snowflake.com/en/LIMITEDACCESS/snowpark-pandas-api/reference/modin/pandas_api/snowflake.snowpark.modin.pandas.DataFrame.join.html) leave it for the Notes section. IMO it's better for users to keep it in the `Parameters` section, since they can tell right away what limitations they're facing, and the Notes section is easy to miss since it's below all the parameters.

Reply:
Added a section in the beginning of the docstring, and parts to each parameter detailing what Snowpark pandas supports.